store/sqlstore/upgrade.go

```diff
@@ -1350,6 +1350,43 @@ func upgradeDatabaseToVersion600(sqlStore *SqlStore) {
 		sqlStore.AlterColumnTypeIfExists("Posts", "FileIds", "text", "varchar(300)")
 	}
 
+	forceIndex := ""
```

Comment:
The other thing is that I am guessing this is meant to go in 6.1 since we are past code complete so we should put the migration in a new `upgradeDatabaseToVersion610` function. You can check https://github.com/mattermost/mattermost-server/pull/18390/files#diff-c7561d15f1d9c1a92942b9ce5d2b1d979c89ffd525a8975f190983220cd6f2af for an example.